tests/bluetooth/host/keys/bt_keys_foreach_bond/prj.conf

```diff
@@ -6,3 +6,6 @@ CONFIG_BT_MAX_PAIRED=4
 CONFIG_ASSERT=y
 CONFIG_ASSERT_LEVEL=2
 CONFIG_ASSERT_VERBOSE=y
+CONFIG_LOG=n
+CONFIG_BT_DEBUG_LOG=n
+CONFIG_TEST_LOGGING_DEFAULTS=n
```

Comment:
You've added these a few places. I am not sure I understand the purpose of this, can you elaborate?

Reply:
Yes, I added that so the logs in files like `bluetooth/host/keys.c` that are used by the tests are not compiled and so there is no need to link the `printk` functions that are used by the logs.

Reply:
Can I bother you to add that to the commit message? At least that is to me not an obvious change, nor is the reasoning part of commit message :)

Reply:
You're right, I'll do it.